dhcp.go

```diff
@@ -66,9 +74,17 @@ func (d dhcpHandler) serveDHCP(w dhcp4.ReplyWriter, req *dhcp4.Packet) {
 	j, err := job.CreateFromDHCP(mac, gi, circuitID)
 	if err != nil {
 		mainlog.With("type", req.GetMessageType(), "mac", mac, "err", err).Info("retrieved job is empty")
+		metrics.JobsInProgress.With(labels).Dec()
+		timer.ObserveDuration()
 		return
 	}
-	go j.ServeDHCP(w, req)
+	go func() {
+		if j.ServeDHCP(w, req) {
+			metrics.DHCPTotal.WithLabelValues("reply", gi.String()).Inc()
```

Comment:
why don't we want to explicitly record total vs successful and thus be able to calculate error'd?

Reply:
We're able to do that with reply.

Reply:
switched to using op=send and type=DHCPOFFER